src/View/User/Includes/_course_tab.html.twig

```diff
@@ -0,0 +1,90 @@
+{% block course_tab %}
+<div class="course-tab-section col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+    <div class="tab-title-text">
+        <p class="tab-title">
```

Comment:
if this is a title, why use the p tag? an h* seems more appropriate

Reply:
Done